hydromt_wflow/wflow.py

```diff
@@ -3147,37 +3147,48 @@ def setup_cold_states(
             self.set_config(option, states_config[option])
 
     # I/O
-    def read(self):
+    def read(
+        self,
+        config_fn: Union[Path, str] = None,
```

Comment:
Do we need this? You can specify the config_fn in the model init or global options in the hydromt model_build.yml config. If we do not then I think we should have something similar for staticgeoms ie defined during model init and keep the current behavior.